ui/apps/platform/cypress/integration/compliance-enhanced/complianceEnhancedScanConfigs.test.js

```diff
@@ -1,13 +1,54 @@
 import withAuth from '../../helpers/basicAuth';
+import { interactAndWaitForResponses } from '../../helpers/request';
 import { getRegExpForTitleWithBranding } from '../../helpers/title';
 import { getHelperElementByLabel, getInputByLabel } from '../../helpers/formHelpers';
+import { navigateWizardForward } from '../../helpers/wizard';
 
 import {
     visitComplianceEnhancedSchedulesFromLeftNav,
     visitComplianceEnhancedScanConfigs,
     complianceEnhancedScanConfigsPath,
 } from './ComplianceEnhanced.helpers';
 
+function interceptAndMockComplianceIntegrations(callback) {
+    const alias = 'complianceIntegrations';
```

Comment:
For your consideration, existing pattern is `compliance/integrations` parallel to request in cypress side panel.